File: EAD_Project/Scripts/billTo.js
$(document).ready(function (e) {
    $('#billTo').click(function () {
            var Users = {
                "Display_Name": $("#Display_Name").val(),
                "User_Name": $("#User_Name").val(),
                "Password": $("#Password").val(),
                "confirm_password": $("#confirm_password").val(),
                "Company_Name": $("#Company_Name").val(),
                "Email": $("#Email").val(),
                "Title": $("#Title").val(),
                "First_Name": $("#First_Name").val(),
                "Middle_Name": $("#Middle_Name").val(),
                "Last_Name": $("#Last_Name").val(),
                "Address_1": $("#Address_1").val(),
                "Address_2": $("#Address_2").val(),
                "Zip": $("#Zip").val(),
                "Country": $("#Country").val(),
                "State": $("#State").val(),
                "Phone1": $("#Phone1").val(),
                "Phone2": $("#Phone2").val(),
                "Mobile_Phone": $("#Mobile_Phone").val(),
                "Fax": $("#Fax").val(),
                "message": $("#message").val(),
                "Shipping": $("#Shipping").val()
            };
            $.ajax({
                type: "POST",
                url: '/Home/Bill_To',
                data: JSON.stringify(Users),
                contentType: "application/json;charset=utf-8",
                processData: true,
                success: function (data, status, xhr) {

                    alert("U have checked out successfully !!! ");
                    location.href = "/Home/NormalUser"
                },
                error: function (xhr) {
                    alert("CheckOut unsuccessful!!! Try Again ");
                }
            });
            return true;
     
    });
});
// Function that validates email address through a regular expression.
//function validateEmail(sEmail) {

//    return /[a-zA-Z0-9]+@[a-zA-Z0-9]+\.[a-zA-Z0-9]+/.test(sEmail);// return /[a-z]+@[a-z]+\.[a-z]+/.test(sEmail);
//}

File: EAD_Project/Models/Bill_To.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace EAD_Project.Models
{
    
    public class Bill_To
    {

        public int Total { get; set; }
        public int UserID { get; set; }
        public int BillNo { get; set; }
        public string Display_Name { get; set; }
        public string User_Name { get; set; }
        public string Password { get; set; }
        public string confirm_password { get; set; }
        public string Company_Name { get; set; }
        public string Email { get; set; }
        public string Roll_Number { get; set; }
        public string Title { get; set; }
        public string First_Name { get; set; }
        public string Middle_Name { get; set; }
        public string Last_Name { get; set; }
        public string Address_1 { get; set; }
        public string Address_2 { get; set; }
        public string Zip { get; set; }
        public string Country { get; set; }
        public string State { get; set; }
        public string Phone1 { get; set; }
        public string Phone2 { get; set; }
        public string Mobile_Phone { get; set; }
        public string Fax { get; set; }
        public string message { get; set; }
        public string Shipping { get; set; }

    }
}